README.md

```diff
@@ -110,6 +110,53 @@ Based on the discussion in [#79](https://github.com/js-reporters/js-reporters/is
 
 Additional properties (not defined here) can be added to the Assertion object.
 
+## Details
+
+For details please check out the [docs](https://github.com/js-reporters/js-reporters/blob/add-examples/docs) and especially the [example](https://github.com/js-reporters/js-reporters/blob/add-examples/docs/example.md) which presents a testsuite and its reporting data based on the standard presented above.
```

Comment:
The paths are wrong, need to remove the extra "add-". Might also be better to use relative links, since absolute links "break" for forks.